server/build/plugins/combine-assets-plugin.js

```diff
@@ -9,19 +11,27 @@ export default class CombineAssetsPlugin {
 
   apply (compiler) {
     compiler.plugin('after-compile', (compilation, callback) => {
-      let newSource = ''
+      const concat = new Concat(true, this.output, '\n')
+
       this.input.forEach((name) => {
         const asset = compilation.assets[name]
+        const sourceMap = compilation.assets[`${name}.map`]
         if (!asset) return
 
-        newSource += `${asset.source()}\n`
-
-        // We keep existing assets since that helps when analyzing the bundle
+        concat.add(name, asset.source(), sourceMap && sourceMap.source())
+        delete compilation.assets[name]
```

Comment:
Here, we don't want to delete old assets. We keep them for better debugging.
Anyway, keeping them won't affect to the app.